### DISCUSSION TITLE
更新后进度集里自定义进度后括号里的数字有一点小问题

### DISCUSSION THREAD

**Discussion by soakup** [WatsonYao/nian_app#114] [Q&A] [answered]
![IMG_20241123_221432](https://github.com/user-attachments/assets/2398253e-2cf1-42af-919a-860cdbae5d55)
第一天18页，第二天28页，应该是19～36页；第三天38页应该是47～74页。
这个填进度是按照每天“读了几页”填的，但是后面的区间好像是直接当成每天“读到第几页”写的。

我是这么理解的，还是应该有其他的理解方法呢？

**Comment by WatsonYao** [comment]
应该是bug，这两天修复后再补充留言。

**Comment by WatsonYao** [comment, marked as answer]
上传了 2024_11_2 版本，撤销了后面括号里的显示。
